### ISSUE TITLE
Implement automated tests for DDEV Quickstart docs

### ISSUE THREAD

**Issue by rfay** [ddev/ddev#6222]
It's really hard to keep all the [Quickstarts](https://ddev.readthedocs.io/en/stable/users/quickstart/) properly updated as the upstream projects change.

### Describe your solution

Implement either
* Automated tests for the quickstarts (include them into the docs as scripts?) or
* Create parallel automated tests that are intended to match each quickstart, and then carefully maintain them.

**Comment by mandrasch** [comment]
👏👏 Will be awesome!

My two cents:

Personally I would argue against docs integration, because this can get messy fast with different versions + the quickstart docs currently are not focused on no-interaction / non-interactive install (which would be needed for automated testing). 

I'm not a technical expert on setting up such tests, but I could imagine that we could put 1 cms in 1 test file and delete/clean between cms versions - if this is meant to be executed with `ddev/github-action-setup-ddev@v1`? Or should it run as some kind of go test? 

Happy to help/contribute!

**Comment by rfay** [comment]
I agree that docs integration will probably be too hard. I'm thinking we could have a single golang test (or bats, or ...) for each quickstart, and just maintain them in parallel with the quickstarts, so by process a change to the quickstart requires an update to the test.

**Comment by rfay** [comment]
This is done, special thanks to @rpkoller !
